src/Symfony/Component/HttpKernel/Config/FileLocator.php

```diff
@@ -75,13 +75,13 @@ public function locate($file, $currentPath = null, $first = true)
                     }
 
                     if (0 === strpos($location, $deprecatedPath) && (null === $currentPath || false === strpos($location, $currentPath))) {
-                        $triggerDeprecation = true;
+                        $deprecation = sprintf('Loading the file "%s" from the global resource directory "%s" is deprecated since Symfony 4.4 and will be removed in 5.0.', $file, $deprecatedPath);
```

Comment:
Alternative: `$triggerDeprecation = $deprecatedPath;` and then use `$triggerDeprecation` later but that seemed less readable.